cpp/test/ome-xerces/xerces.cpp

```diff
@@ -156,6 +162,15 @@ TEST_P(XercesTest, EmptyDocumentCreateElement)
   root.appendChild(e);
 }
 
+TEST_P(XercesTest, EmptyDocumentCreateElementNS)
+{
+  xml::dom::Document document(ome::xerces::dom::createEmptyDocument("http://example.com/test/namespace1", "root"));
+  ASSERT_TRUE(document);
+  xml::dom::Element e(document.createElementNS("http://example.com/test/namespace2", "test"));
+  xml::dom::Element root(document.getDocumentElement());
+  root.appendChild(e);
+}
```

Comment:
We could verify that the namespace is correct and all the elements are inserted correctly.  The main purpose is to ensure that the wrappers are valid and don't throw exceptions on any of the method calls, which doesn't require any assertions.